Makefile

```diff
@@ -10,10 +10,10 @@ all: dollarskip
 install: dollarskip
 	$(STRIP) dollarskip
 	install -d $(DESTDIR)$(PREFIX)/bin
-	install -m 755 dollarskip $(DESTDIR)$(PREFIX)/bin/\$
+	install -m 755 dollarskip $(DESTDIR)$(PREFIX)/bin/\$$
```

Comment:
```suggestion
	install -m 755 dollarskip $(DESTDIR)$(PREFIX)/bin/$$
```
I don't think it actually matters, but it makes more sense at least to me.

Reply:
I really don't know if this will break compatibility on different systems. If it doesn't then I'm all for this change

Reply:
I'm pretty sure `$$` is used for escaping `$` on all versions of make, but I have never checked.

Reply:
I'm pretty sure that POSIX requires this, I'm also pretty sure that POSIX does not require `sh` to accept broken input.

Reply:
Does make expand variables inside `'` (single quotes)? Bash doesn't.
Otherwise looks good to me.

Reply:
`make` does not parse single quotes.

Reply:
so `install -m 755 dollarskip '$(DESTDIR)$(PREFIX)/bin/$$'` is ok.
what do you think @CleanMachine1?

Reply:
Yes however are we sure this works on Linux

Reply:
why wouldnt it?